pkg/api/defaults_test.go

```diff
@@ -116,14 +116,18 @@ func TestSetDefaults(t *testing.T) {
 			},
 		},
 		{
-			name: "preserve flags",
+			// TODO: Once the operatorflags are changed from strings to consts, we can correctly
+			// modify a single flag, until then it's hardcoded
+			name: "merge flags",
```

Comment:
Do we want to test the other case as well (`doc.OpenShiftCluster.Properties.OperatorFlags == nil`, set them)?